src/github.js

```diff
@@ -199,21 +197,21 @@ ns.bumpFiles = function (bump) {
     var v = versiony.from('package.json').indent(jsonIndent);
 
     switch (bump) {
-        case 'major':
-            v.newMajor();
-            break;
+    case 'major':
```

Comment:
Did the new lint complain about indented `case` within `switch`? I've always preferred them indented myself...